implementation/orchestration-to-choreography/lambda/choreography/quoteAggregator.py

```diff
@@ -0,0 +1,20 @@
+import boto3
+import json
+import logging
+import uuid
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+dynamo = boto3.resource('dynamodb')
+
+def lambda_handler(event, context):
```

Comment:
Looks like the aggregator is Not aggregating but simply inserting into the table as individual items.
See Gregor's example here: https://www.enterpriseintegrationpatterns.com/ramblings/loanbroker_stepfunctions_pubsub.html#aggregator

Reply:
Implemented a simple aggregation logic as section 5 in Gregor's example. To keep the solution simple and concise, I didn't introduce the more sophisticated solutions in section 6 and Bonus part.